spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/ZuulMetricsInitializer.java

```diff
@@ -0,0 +1,33 @@
+package org.springframework.cloud.netflix.zuul;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+
+import com.netflix.zuul.monitoring.CounterFactory;
+import com.netflix.zuul.monitoring.TracerFactory;
+
+import lombok.extern.apachecommons.CommonsLog;
+
+@CommonsLog
+public class ZuulMetricsInitializer implements ServletContextListener {
```

Comment:
I'd rather have once class that does this. Any reason to not include this in `ZuulFilterInitializer`?